calico/reference/cni-plugin/configuration.md

```diff
@@ -154,6 +154,28 @@ The following options allow configuration of settings within the container names
 }
 ```
 
+### Readiness Gates
+
+The following option makes CNI plugin wait for specified  endpoint/s to be ready before configuring pod networking.
+
+* `readiness_gates`
+
+This is an optional parameter.
+
+Example CNI config:
+
+```json
+{
+    "name": "any_name",
+    "cniVersion": "0.1.0",
+    "type": "calico",
+    "ipam": {
+        "type": "calico-ipam"
+    },
+    "readiness_gates": ["http://localhost:9099/readiness"],
```

Comment:
Maybe we can put more than one gate in the example to make it clear it can take many:
```
    "readiness_gates": ["http://localhost:9099/readiness", "http://localhost:8888/status"],
```

Reply:
Modified.